docs/src/full tutorials/Spam Detection with RNNs/SMS.jl

```diff
@@ -0,0 +1,180 @@
+# # SMS Spam Detection with RNNs
+
+# In this tutorial we use a custom RNN model from Flux with MLJFlux to classify text messages as spam or ham. We will be using the [SMS Collection Dataset](https://www.kaggle.com/datasets/uciml/sms-spam-collection-dataset) from Kaggle.
+
+using Pkg     #src
+Pkg.activate(@__DIR__);     #src
+Pkg.instantiate();     #src
+
+# ### Basic Imports
+using MLJ
+using MLJFlux
+using Flux
+using CSV               # Read data
+using DataFrames        # Read data
+using ScientificTypes   # Type coercion
+using WordTokenizers    # For tokenization
+using Languages         # For stop words
+
+
+# ### Reading Data
+df = CSV.read("./sms.csv", DataFrame);
+
+# Display the first 5 rows with DataFrames
+first(df, 5) |> pretty
+
```

Comment:
FYI. I used to use `pretty` a lot but maybe this is not really necessary as  DataFrames display is pretty sophisticated now.

Reply:
Agree. I like DF display more.